src/SplashScreen/SplashFooter/SplashFooter.tsx

```diff
@@ -86,7 +86,7 @@ function SplashFooter() {
                 <img src={logo} alt={'white pr logo'} />
               </a>
             </StyledLogo>
-            <p>
+            <p style={{ paddingBottom: 'calc(4vw - 10px)'}}>
```

Comment:
We're trying to stay away from inline styles with the `style` prop. Could you change this to use the `Text` component from Chakra. 

Update line 23 to import Text like this

```
import { Flex, Text } from '@chakra-ui/core';
```

```suggestion
            <Text paddingBottom='calc(4vw - 10px)'>
```